modules/api/php/endpoint.class.inc

```diff
@@ -72,4 +72,33 @@ abstract class Endpoint implements RequestHandlerInterface
 
         return $handler->handle($request);
     }
+
```

Comment:
Updating the genomic browser to use React shouldn't involve modifying the LORIS  API.